### ISSUE TITLE
Interactive playground should localize mouse shortcuts

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#21512]
With --locale=ko and de, notice the keybindings are localized but the mouse ones are not.

![image](https://cloud.githubusercontent.com/assets/2193314/23374065/87c53854-fcd7-11e6-8260-886e83ee31d6.png)

![image](https://cloud.githubusercontent.com/assets/2193314/23374094/a34b9ff0-fcd7-11e6-965f-0a1818d7f4ba.png)

**Comment by chrmarti** [comment]
@dbaeumer I understand you were getting infrastructure in place that would allow us to translate the playground document?

**Comment by dbaeumer** [comment]
Yes and no. What we did is to move all translation tooling into the VS Code repository. The translation itself is happening on Transifex. So we need to extend the tooling to convert the md file into a xliff file and push and pull it from Transifex.

The corresponding code to talk to Transifx is in i18n.ts in the build folder in VS Code. @michelkaporin how complicated would it be to support the md file.

**Comment by michelkaporin** [comment]
If we do localise keybindings, should not we also localise the whole playground text? 
Personally, I don't see a reason for writing separate Markdown parsing to XLF format (the one we use for pushing to Transifex) just for the mouse bindings. Maybe there is the same way to dynamically replace those, as it's done with keybindings (e.g. `kb(cursorColumnSelectDown)` is replaced during runtime)? This would be the quick fix. Although in the long run we should translate everything.

As an approach of translating the whole document I see two options:
1. Generating `.md` files in code, with `nls.localize()` calls. This would not require adding new logic around translations using Transifex. 
2. Writing separate `.md -> .xlf -> .md` parser just for the Markdown files. In my opinion this is not the best approach, however if we plan to have more `.md` documents to be translated (e.g. upgrade notes), then we can look into it, as this would simplify writing the code as described in the first option.

I would prefer the first approach at this point of time.
